### ISSUE TITLE
when addressToFIPS returns empty array - error

### ISSUE THREAD

**Issue by talamihg** [uscensusbureau/citysdk#313]
```
    //Check for an address object
    if("address" in request) {
        //We have address - but do we have lat/lng?
        if(!("lat" in request) || !("lng" in request)) {
            //We have the address but no lat/lng - parse it and re-call
            this.addressToFIPS(request.address.street, request.address.city, request.address.state, function(response) {
                //Take the first matched address
                request.lat = response[0].coordinates.y;
                request.lng = response[0].coordinates.x;

                //Attach this "matched address" to the request address object so the user knows what we're using
                request.address.addressMatch = response[0];

                CitySDK.prototype.sdkInstance.modules.census.APIRequest(request, callback);
                return;
            })
        }
    }
```

if the function addressToFIPS cannot find the address it will return an empty array. The line

```
request.lat = response[0].coordinates.y;
```

throws an error

```
citysdk.census.js:1522 Uncaught TypeError: Cannot read property 'coordinates' of undefined
```

**Comment by talamihg** [comment]
This might be using code from out of date sources. I realize that I am using Release 1.1 from 
https://cdn.rawgit.com/uscensusbureau/citysdk/Release1.1/js/citysdk.census.js

However the Getting Started guides show to use this. 
https://uscensusbureau.github.io/citysdk/gettingstarted.html

**Comment by logantpowell** [comment]
Hi @talamihg very sorry for the delay on this. I'm not sure how I missed this issue. Did you get the updated release?

We will correct the getting started guides as soon as we get back on our feet (currently sans developer).

**Comment by talamihg** [comment]
Hi, I did not get the update, but I was able to come up with a workaround that is just for my own purpose.  I was able to add `.fail(function(){ ... })` to the ajax call `CitySDK.prototype.jsonpRequest = function(url){...}` I am now handling any errors due to network connectivity or due to a failed geocoded response. 

I am not requesting any more help on this matter. So at this point, it sounds like this issue is resolved in 2.0?? I'll let you update this issue to resolved or closed. 

Thanks for checking in on this issue!
